test/verify/check-host-switching

```diff
@@ -0,0 +1,423 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import subprocess
+
+import parent
+from testlib import *
+from machine_core.constants import TEST_OS_DEFAULT
+
+
+@skipImage("Introduced in #13663", "rhel-8-2-distropkg")
```

Comment:
This isn't necessary, and wrong -- not every test class which uses this as a helper should automatically be skipped.